jetty-ee9/jetty-ee9-nested/pom.xml

```
@@ -9,7 +9,7 @@
   </parent>
   <artifactId>jetty-ee9-nested</artifactId>
   <name>EE9 :: Nested</name>
-  <description>The jetty core handler adapted to legacy ee9 handler.</description>
+  <description>Jetty core handler adapter for EE9 (jakarta.servlet 5.0) environments.</description>
```

This is not core, nor is it handler specific.

```suggestion
  <description>Jetty EE9 (jakarta.servlet) nested implementation.</description>
```